src/events/acceleration.js

```diff
@@ -203,6 +244,18 @@ p5.prototype._handleMotion = function() {
     }
     old_max_axis = new_max_axis;
   }
+  var deviceShaken = this.deviceShaken || window.deviceShaken;
+  if (typeof deviceShaken === 'function') {
+    var accelerationChange = {};
```

Comment:
@lmccart Ok. So just create variables accelerationChangeX, Y, and Z instead of deviceShaken() and setShakeThreshold()?